_posts/2019-01-20-artsy-engineering-hiring.md

```diff
@@ -0,0 +1,300 @@
+
+The intention of in-person coding challenges is to verify that the engineer can "actually write code." This
+strategy puts excessive pressure on the candidate to perform in front of an audience. This is usually not a good
+reflection of what the candidate would be doing if they were hired. Sometimes it **is** a reflection of the
+stressful conditions on the team, and the act of applying pressure to the candidate is intentional, to measure
+their ability to handle it. In either case, we don't feel like this is how we want to measure engineers; it just
+don't reflect reality.
+
+### Whiteboard interviews
+
+One intention of whiteboard interviews is to reduce the stress on the candidate, because they don't have to worry
+about code syntax while under a microscope. These types of interviews still lead to stressful conditions, though,
+and they don't provide a good measure of what makes a great teammate or even a great developer. Again, sometimes
+the pressure is intentional, to see how the candidate reacts.
+
+It can be very difficult to find a problem that is succinct enough for a whiteboard exercise but still reflective
+of the work the candidate will actually be doing on the job. The ability to write an algorithm to search a binary
+tree might be reflective of whether a candidate has a traditional Computer Science degree, but doesn't say much
+about their ability to build complex interfaces or streamline performance. Questions like this can eliminate
+excellent developers who took a non-traditional approach to their knowledge building but are still highly capable.
+
+### Sample code
+
+Sometimes a company will request a code sample from candidates - after all, what shows off their ability to code
+better than their actual code? The downfall of this strategy is that it eliminates developers who don't have code
+they can share. Many great engineers work for closed-source companies; many great engineers have family
+responsibilities that prevent them from contributing to open-source at night.
+
+It is also important to consider the insularity and biases that exist in the open source community that can make
+contributing more difficult for developers from underrepresented groups. A study published in the PeerJ Computer
+Science journal found that women’s contributions to open source projects were accepted more frequently than men’s
+contributions when the gender of the contributor was unknown. However, when the gender of the contributor was
+apparent, men's contributions were
+[accepted more frequently than women's](https://code.likeagirl.io/gender-bias-in-open-source-d1deda7dec28).
+
+### Take-home challenges
+
+The most recent trend in hiring is the take-home exercise. The goal is honorable - have the candidate produce code
+on their own time, so they aren't overwhelmed with the pressure of an audience. We've found that it's not fair to
+require this early in the hiring process, and it's not informative later in the process.
```

Comment:
> We've found that it's not fair to require this early in the hiring process, and it's not informative later in the process.

> This strategy also assumes...

With a paragraph break and the "also" here it feels like the previous sentence doesn't resolve with _why_ its not informative later in the process, like it was cut a bit short.